test/integration/server-selection/server_selection.spec.test.ts

```diff
@@ -0,0 +1,20 @@
+import * as path from 'path';
+
+import { loadSpecTests } from '../../spec';
+import { runUnifiedSuite } from '../../tools/unified-spec-runner/runner';
+
+describe.skip('Server Selection Unified Tests (Spec)', function () {
```

Comment:
Is skipping here intentional? If yes we could leave a todo with a ticket to unskip it.